docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,174 @@
+in ``fault_ever`` place the module into safe mode. The module remains in safe mode until the user clears ``fault_ever``. When ``fault_latching`` is set to 0, 
+the module remains in safe mode until all power safety parameters are back in their safe operating range, and ``fault_now`` returns to 0.
+
+Internal Programmable Voltage Detector (PVD)
+------------------------------------------------
+Each module's microcontroller has an internal PVD that monitors the microcontroller's Vdd voltage level. In the event that Vdd drops below the pre-programmed threshold 
+value (~2.8V), the module is forced to stop spinning. The module will only be allowed to resume normal operation when Vdd has risen back above the voltage threshold (~2.9V).
+
+.. _slew_rate:
+
+Slew Rate
+===========
+Your module's slew rate defines the maximum change in control voltage per second. Slew rate works as a form of software filter that ensures that your module does not 
+attempt to change speeds faster than is safe.
+
+There are two parameters in the :ref:`Brushless Drive client <brushless_drive>` that affect your module's slew rate limiting:
+
+* ``slew_volts_per_second`` defines the actual slew rate in volts per second applied to voltage commands. This defines the strength of the filter's impact.
+* ``slew_enable`` determines whether or not ``slew_volts_per_second`` is actually applied to voltage commands. If ``slew_enable`` is false, then there will not be slew limiting applied.
+
+The following examples are meant to illustrate the effect that slew limiting has on module control. All examples start with the module in coast followed by a voltage step command to 2V.
+
+1. Slew limiting disabled
+
+.. image:: ../_static/manual_images/safety/slew_rate_off.png
+    :scale: 80%
+
+2. Slew limit enabled at 10V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_10vps.png
+    :scale: 80%
+
+3. Slew limit enabled at 5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_5vps.png
+    :scale: 80%
+
+4. Slew limit enabled at 2.5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_2_5vps.png
+    :scale: 80%
+
+5. Slew limit enabled at 1V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_1.png
+    :scale: 80%
+
+As you can see, the slew limit parameter helps protect against sudden changes in commanded voltage. This can be especially useful in :ref:`noisy/vibration heavy 
+environments <vibration_and_jittering>`. The amount of slew filtering that is necessary (if any) is completely application specific, and will require manual tuning.
+
+Brushless Drive Current Limiter
+====================================
+:ref:`Brushless Drive's <brushless_drive>` ``motor_I_max`` defines that maximum allowable drive current. In general, this current limit protects your module against extreme step commands, and ensures that the current moving through the motor is always in a safe region.
```

Comment:
A couple things content-wise on this first bit. You do say it and I know that you know it, but it may be worth driving home because people often get confused on this: this is the maximum allowable motor current, not supply current. They are not necessarily the same.

About the second sentence, it does protect us against extreme step commands, and specifically it's meant to protect the module's electronics from current spikes that may permanently damage them. It's also useful for protecting them during stalls, which don't really involve an actual step command, but do lead to a huge surge in current. So maybe just don't frame it as just for steps, though they are part of it.